README.md

```diff
@@ -37,13 +37,32 @@ OpenSearch Dashboards Reports integration with [Notifications](https://github.co
 
 ## Troubleshooting
 
-#### Fail to launch Chromium
+### Fail to launch Chromium
 
 There could be two reasons for this problem
 
 1. You are not having the correct version of headless-chrome matching to the OS that your OpenSearch Dashboards is running. Different versions of headless-chrome can be found [here](https://github.com/opensearch-project/dashboards-reports/releases/tag/chromium-1.12.0.0)
 
 2. Missing additional dependencies. Please refer to [additional dependencies section](./dashboards-reports/rendering-engine/headless-chrome/README.md#additional-libaries) to install required dependencies according to your operating system.
+
+### Missing Font Dependencies
+Chromium may not have all of the dependencies you may require to be able to view all of the content of your reports.
+
+If you are using a CentOS/RHEL system, install the following packages:
+* [`ipa-gothic-fonts`](https://centos.pkgs.org/8/raven-x86_64/ipa-gothic-fonts-003.03-15.el8.noarch.rpm.html)
+* [`xorg-x11-fonts-100dpi`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-fonts-100dpi-7.5-9.el7.noarch.rpm.html)
+* [`xorg-x11-fonts-75dpi`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-fonts-75dpi-7.5-9.el7.noarch.rpm.html)
+* [`xorg-x11-utils`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-utils-7.5-23.el7.x86_64.rpm.html)
+* [`xorg-x11-fonts-cyrillic`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-fonts-cyrillic-7.5-9.el7.noarch.rpm.html)
+* [`xorg-x11-fonts-Type1`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-fonts-Type1-7.5-9.el7.noarch.rpm.html)
+* [`xorg-x11-fonts-misc`](https://centos.pkgs.org/7/centos-x86_64/xorg-x11-fonts-misc-7.5-9.el7.noarch.rpm.html)
+* [`fontconfig`](https://www.freedesktop.org/wiki/Software/fontconfig/)
+* [`freetype`](https://freetype.org/)
+
+If you are using a Ubuntu/Debian system, install the following packages:
+* [`fonts-liberation`](https://packages.debian.org/search?keywords=fonts-liberation)
+* [`libfontconfig1`](https://packages.debian.org/sid/libfontconfig1)
```

Comment:
maybe also provide the yum/apt install -y command like in https://github.com/opensearch-project/dashboards-reports/blob/main/dashboards-reports/rendering-engine/headless-chrome/README.md#additional-libaries?

Reply:
Added a link to that command since the packages listed are actually all the same. Also added ipa-gothic-fonts to that section and linked the centos 7 version instead (ended up finding it)